### PR TITLE
First Iteration of Availability Front End under "User Profile" Created

### PR DESCRIPTION
**Pull request by vinitbytheminute**
Schedule Availability
1. Created Availability section in the sidebar
2. Added When2Meet to Availability Section
3. Added Save and Update Button

### CODE REVIEW

**Comment by amanmibra** [comment]
Good work, but move the Availability down to the bottom of the list of 'User Information', 'Edit User ..', etc. Also, I would reword it to 'Edit User Availability'. 

Moreover, change the coloring of the buttons to the default coloring (please refer to the other panels on the coloring), and also, why do you have a 'Save' *AND* a 'Update' button. That makes no sense, just have a 'Save' button like the others. Consistency is key.

**Comment by amanmibra** [comment]
Check out Rikki's branch and PR #2 where he has the key for his Availability component in the sign up component. It looks nice and you may not need to do anything for the key if you like it.

**Comment by vinitbytheminute** [comment]
Thank you for the comment. I will go ahead and check that out Rikki's version
